src/scripts/store-daily-stats.ts

```diff
@@ -0,0 +1,47 @@
+import Knex from "knex";
+import kmqKnexConfig from "../config/knexfile_kmq";
+
+const storeDailyStats = async () => {
+    const db = Knex(kmqKnexConfig);
+
+    if (!(await db.schema.hasTable("daily_stats"))) {
```

Comment:
This should be a migration

Reply:
Your documentation in DEVELOPER.md was very helpful, thanks!